ads/performancenative.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {validateData, writeScript} from '../3p/3p';
+
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+ 
+
+	
+export function performancenative(global, data) {
+	
+  validateData(data, ['section_key', 'content_scope','amp_page']);
+  global.section_key = data.section_key;
+  global.content_scope = data.content_scope;
+  global.amp_page = data.amp_page;
+  global.amp_content = "";
+  
+  writeScript(global, 'https://api.performancenative.com/assets/pn.js');
```

Comment:
Do you need this to be a synchronous write? If not could you please use `loadScript()` instead?